src/android/app/src/main/res/menu/menu_landscape_screen_layout.xml

```diff
@@ -23,6 +19,15 @@
             android:id="@+id/menu_screen_layout_hybrid"
             android:title="@string/emulation_screen_layout_hybrid" />
 
+        <item
+            android:id="@+id/menu_screen_layout_custom"
+            android:title="@string/emulation_screen_layout_custom" />
+
     </group>
 
+    <item
+        android:id="@+id/menu_adjust_custom_layout"
+        android:title="@string/emulation_adjust_custom_layout"
+        android:enabled="false" />
```

Comment:
Instead of using this item to tell this to the user, I would instead use an Android toast notification similar to the "Saved Settings" popup.

Reply:
Toasts disappear, don't they? When are you imagining it appearing, when the menu is first opened?

Reply:
I would suggest that it appears after the custom layout option is select